src/EditorFeatures/TestUtilities/Workspaces/TestHostProject.cs

```diff
@@ -124,6 +124,9 @@ public string FilePath
             }
         }
 
+        internal void OnProjectFilePathChanged(string filePath)
+            => _filePath = filePath;
```

Comment:
?

Reply:
CodeStyle layer testing is based on adding an explicit auto-generated editorconfig into the input project. Due to https://github.com/dotnet/roslyn/issues/41940, we are currently forced to generate a rooted editorconfig, and in turn forced to have the project and documents also have rooted file paths so that the editorconfig is applied to them. This change was done to ensure that test framework can change the input project path to make it a rooted one. This can likely be reverted once https://github.com/dotnet/roslyn/issues/41940 is fixed OR if we move all our tests to Microsoft.CodeAnalysis.Testing, whichever happens first.